components/support/sync-telemetry/metrics.yaml

```
@@ -4,6 +4,47 @@
 
 $schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
 
+sync:
+  sync_uuid:
+    type: uuid
+    description: >
+      Unique identifier for this sync, used to correlate together individual pings for data types that
+      were synchronized together (history, bookmarks, logins). If a data type is synchronized by itself via
+      the legacy 'sync' API (as opposed to the Sync Manager), then 'syncUuid' will not be set on the corresponding ping.
```

```suggestion
      the legacy 'sync' API (as opposed to the Sync Manager), then this field will not be set on the corresponding ping.
```

Note that `syncUuid` is the label of the generated object in Kotlin, the field is called `sync_uuid`. We can also just not mention it by name, here.